src/wrp-c.h

```diff
@@ -14,8 +14,9 @@
  * limitations under the License.
  *
  */
-#include <sys/types.h>
+#include <stdbool.h>
```

Comment:
Thanks, I missed that!
On Jun 24, 2016 5:24 PM, "Khem Raj" <notifications@github.com> wrote:

> In src/wrp-c.h
> <https://github.com/Comcast/wrp-c/pull/5#discussion_r68479643>:
>
> > @@ -14,8 +14,9 @@
> >   * limitations under the License.
> >   *
> >   */
> > -#include <sys/types.h>
> > +#include <stdbool.h>
>
> For stdbool you need to demand c99 from compiler
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Comcast/wrp-c/pull/5/files/13a5562a3902043247f6975c94c33be8cab6ff9b#r68479643>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAqpyS-K8x2U5dToLmIMrTi16Z_cC093ks5qPHVKgaJpZM4I-N-7>
> .
>